pages/common/gulp.md

```diff
@@ -0,0 +1,11 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
```

Comment:
I was going to suggest mentioning the gulpfile in the description of the second comment, but here works too :)